package.json

```diff
@@ -1,13 +1,24 @@
 {
   "name": "bcgov-digital-marketplace",
+  "jest": {
+    "verbose":true,
+    "moduleDirectories": [
+      "node_modules",
+      "src"
+  ],
+    "moduleNameMapper": {
+      "front-end/(.*)": "<rootDir>/src/front-end/typescript/$1",
+      "shared/(.*)": "<rootDir>/src/shared/$1"
+  }
+  },
   "version": "1.8.2",
   "description": "Digital Marketplace",
   "main": "index.js",
   "scripts": {
     "start": "npm run back-end:start",
     "front-end:lint": "npx tslint -c tslint.json src/front-end/*.ts src/front-end/**/*.ts && echo Lint Successful\n",
     "front-end:typecheck": "npx tsc --project src/front-end/typescript --noEmit && echo Typecheck Successful\n",
-    "front-end:test": "node tests/front-end/index.js",
+    "front-end:test": "jest",
```

Comment:
If this PR is going to drop runner support for existing front-end harness, perhaps it would be a good idea to delete [the `tests/front-end` folder](https://github.com/bcgov/digital_marketplace/tree/development/tests/front-end).